server/api/subteam/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class SubTeam(models.Model):
+    subteam_id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=254)
+    # to add in later like chantelle said in issue
+    # team_id = models.ForeignKey()
```

Comment:
Team is in main now so you can make team_id a field :) You'll have to import the team model into its file

Reply:
yup added